package-lock.json

```diff
@@ -9,6 +9,7 @@
         "@antora/cli": "~3.0",
         "@antora/site-generator-default": "~3.0",
         "@antora/site-generator-ms": "git+https://gitlab.com/opendevise/oss/antora-site-generator-ms#as-extension",
+        "@asciidoctor/tabs": "^1.0.0-beta.5",
```

Comment:
I guess my only thing - are we cool with implementing something that's still in beta and have we tested appropriately?

Reply:
Good question! @mojavelinux what remains for you to call this a 1.0.0 release?
Happy to put some resources into testing with a specific plan against our estate.

Reply:
To be honest, I was kind of waiting for Couchbase to migrate to Asciidoctor Tabs as that could tease out any bugs that may have been introduced by setting up the project. If you can verify that it's ready to go, that would definitely give me the verification I need to call it 1.0.0.

Keep in mind that beta means "no breaking changes, only bug fixes and new features". So a beta is very stable. It's only really an alpha I would question. But still, we might as well get it over the line.

Reply:
Thanks @mojavelinux, I wondered if that might be the case!
From my POV, the docs were very clear and smooth. The main things I noted in the upgrade process were:
* the preview extension loading, which you submitted the fix for (but that's just tech debt on our part)
* overriding the CSS.  (I ended up deleting all our existing CSS, and copying/customizing the bundled `@asciidoctor/tabs.css` if you have any comments on https://github.com/couchbase/docs-ui/pull/165 do shout). But again, I imagine most/all users will be incorporating this fresh, rather than upgrading from the original script.
* the repo README.adoc should link directly to the excellent Antora page.

@sarahlwelton can you have a quick look at your sections for any issues? (and let me know your  thoughts on what additional testing will reassure you best)